src/main/java/com/javadiscord/javabot/service/help/UnreserveCommandHandler.java

```diff
@@ -24,7 +24,7 @@ public ReplyAction handle(SlashCommandEvent event) {
 			var reasonOption = event.getOption("reason");
 			String reason = (reasonOption == null) ? null : reasonOption.getAsString();
 			channelManager.unreserveChannelByUser(channel, owner, reason, event);
-			return event.deferReply(true);
+			return event.deferReply(false);
```

Comment:
Why are you setting this to false? We want the response from unreserving to be ephemeral, right?